app/src/main/AndroidManifest.xml

```diff
@@ -34,11 +34,9 @@
         <!-- Auth UI demo -->
         <activity
             android:name=".auth.AuthUiActivity"
-            android:theme="@style/AppTheme.NoActionbar"
             android:label="@string/name_auth_ui"/>
```

Comment:
I know about that. If I remove theme setting in configuration it still doesn't  work.